leakcanary-android-core/src/main/java/leakcanary/internal/tv/TvToast.kt

```diff
@@ -28,15 +29,18 @@ internal object TvToast {
     activity: Activity,
     text: CharSequence
   ): Toast {
-    val toast: Toast = Toast.makeText(activity, text, Toast.LENGTH_LONG)
+    val inflater = LayoutInflater.from(activity)
+    val toast = Toast(activity)
 
-    // Custom toast views are deprecated from API level 30 on, see
-    // https://developer.android.com/reference/android/widget/Toast#getView()
-    if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.Q) {
-      val textView = toast.view!!.findViewById<TextView>(android.R.id.message)
-      textView.setCompoundDrawablesWithIntrinsicBounds(R.drawable.leak_canary_icon, 0, 0, 0)
-      textView.compoundDrawablePadding =
-        activity.resources.getDimensionPixelSize(R.dimen.leak_canary_toast_icon_tv_padding)
+    val iconSize = activity.resources.getDimensionPixelSize(
+      R.dimen.leak_canary_toast_icon_size
+    )
+    toast.apply {
+      setGravity(Gravity.CENTER_VERTICAL, 0, -iconSize)
```

Comment:
It's not obvious from the code, how the negative `iconSize` will look on the screen. Mind adding a comment here on what it will look like?

Reply:
I completely removed `iconSize` because it introduced confusion and was not necessary. This toast will appear in the center so we don't need to add yOffset. Thanks for noticing 🙏